### PR TITLE
allow for ga prefix

### PR DESCRIPTION
**Pull request by collin**
@spra85

### CODE REVIEW

File: src/videojs.ga.coffee [hunk 1]
```diff
@@ -112,14 +112,20 @@ videojs.plugin 'ga', (options = {}) ->
 
   sendbeacon = ( action, nonInteraction, value ) ->
     # console.log action, " ", nonInteraction, " ", value
+    send = if options.gaPrefix
+      "#{options.gaPrefix}.send"
+    else
+      'send'
     if window.ga
-      ga 'send', 'event',
+      ga send, 'event',
```
Comment (collin) [inline src/videojs.ga.coffee]: Send with prefix is one is specified.

File: src/videojs.ga.coffee [hunk 2]
```diff
@@ -112,14 +112,20 @@ videojs.plugin 'ga', (options = {}) ->
...
         'eventCategory' 	: eventCategory
         'eventAction'		  : action
         'eventLabel'		  : eventLabel
         'eventValue'      : value
         'nonInteraction'	: nonInteraction
     else if window._gaq
+      if options.gaPrefix
+        throw new Error('videojs.ga plugin does not support a gaPrefix with the `_gaq` implementation.')
```
Comment (collin) [inline src/videojs.ga.coffee]: Not sure how to apply the prefix for `_gaq`, so going to throw an error.

**Comment by spra85** [comment]
:+1:

**Comment by spra85** [comment]
@collin, can you provide a code snippet on how you'd pass in the option?  Is it just part of the standard VideoJS plugin config object

**Comment by collin** [comment]
Yeah, it'll go here: https://github.com/theonion/videohub-player/blob/master/src/player.js#L169-L174

```
  this.player.ga({
    eventActionPrefix: 'video_',
    percentsPlayedInterval: 25,
    eventCategory: 'Video:' + window.channelName,
    eventLabel: window.videoUrl,
    gaPrefix: <prefix goes here>,
  });
```

Don't have exact idea on where the prefix will be built out.

**Pull request merged**